pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>3.2.0</version>
+        <version>2.6.0</version>
```

Comment:
It's really strange to see downgrade of version
what is the reason for such decision?

Reply:
there were problems with the JPA manager for the Book class , but I realized what I was doing wrong